spec/spec_helper.rb

```diff
@@ -1,2 +1,6 @@
 RSpec.configure do |config|
```

Comment:
The gem is configured within the gem's test suite. Otherwise all of this information would need to be set within each of the tests. Gem configurinception.